Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5667,7 +5667,7 @@ StorageAPIEstimateEnabled:
 
 StorageAccessAPIEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
I wanted to retain the toggle for this one.